lib/v3/script.js

```diff
@@ -180,45 +220,13 @@ class ScriptParser extends EventEmitter {
                     const eventItem = this.parseEventDeclaration(node);
 
                     this.emit('event', eventItem, parseContext);
-
-                    return;
                 }
+
+                return;
             }
 
             if (node.type === 'VariableDeclaration' && parseContext.scopeType !== SCOPE_MARKUP) {
-                const variables = parseVariableDeclaration(node);
-
-                variables.forEach(variable => {
-                    if (level === 0) {
-                        this.emit('data', variable, parseContext, 'private');
-                    }
-
-                    if (variable.declarator.init) {
-                        const idNode = variable.declarator.id;
-                        const initNode = variable.declarator.init;
-
-                        // Store top level variables in 'identifiers'
-                        if (level === 0 && idNode.type === 'Identifier') {
-                            this.identifiers[idNode.name] = variable.declarator.init;
-                        }
```

Comment:
This is where the identifiers were added, but this path is not taken for exported nodes.